com.github.diamondburned.gtkcord4.yml

```diff
@@ -0,0 +1,232 @@
+id: com.github.diamondburned.gtkcord4
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gtkcord4
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+
+modules:
+  - name: gtkcord4
+    build-commands:
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+    build-commands:
+      - go install -v -tags libadwaita -mod=vendor ./...
+      - install -Dm644 .nix/$FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 internal/icons/svg/logo.svg /app/share/icons/hicolor/scalable/apps/gtkcord4.svg
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/diamondburned/gtkcord4
+        tag: v0.0.6
+
+      - type: file
+        path: com.github.diamondburned.gtkcord4.metainfo.xml
+        sha256: 04a76929f8b1e5070ca5c37be203fb2e869178e5aa836d53629d822e23e9e3f1
+
+      # Workaround for Go modules generated by github.com/dennwc/flatpak-go-mod
+      - type: file
+        path: modules.txt
+        dest: vendor
+
+      - type: archive
```

Comment:
just a quick note: usually the generated manifest from these tools is "included" instead of being embedded. This allow just running the script to regenerate.
Something like
```yaml
    - path_to_file.json
```

Reply:
I forgot to add modules.txt earlier. It's generated before building the flatpak. The `dest` key is needed. [source](https://github.com/flatpak/flatpak-builder-tools/blob/c60dcb2dcd504dd87491d0b6cca8317d624b51b3/go-modules/README.md)